akka-persistence-testkit/src/test/scala/akka/persistence/testkit/scaladsl/UnpersistentEventSourcedSpec.scala

```diff
@@ -53,11 +53,7 @@ object UnpersistentEventSourcedSpec {
               context.log.debug2("Recovered state for id [{}] is [{}]", id, state)
               recoveryDone ! Done
           }
-          .snapshotWhen {
-            case (_, SnapshotMade, _) => true
-            case _                    => false
-          }
-          .withRetention(RetentionCriteria.snapshotEvery(numberOfEvents = 3, keepNSnapshots = 2))
+          .withRetention(RetentionCriteria.snapshotEvery(numberOfEvents = 2, keepNSnapshots = 2))
```

Comment:
This had to change because now we do not allow anymore to combine `snapshotWhen` and `snapshotEvery`

Reply:
Interesting, this means we will loose the `keepNSnapshots = 2` of the current api usage.

Reply:
Not sure it is important to keep it snapshotting at the exact same points here though, so maybe fine.